scripts/ci-upload-binaries.sh

```diff
@@ -96,8 +96,11 @@ function kotsbin() {
     local kots_version=
     kots_version=$(make print-KOTS_VERSION)
 
-    local kots_override=
-    kots_override=$(make print-KOTS_BINARY_URL_OVERRIDE)
+    local kots_url_override=
+    kots_url_override=$(make print-KOTS_BINARY_URL_OVERRIDE)
+
+    local kots_file_override=
+    kots_file_override=$(make print-KOTS_BINARY_FILE_OVERRIDE)
 
     # check if the binary already exists in the bucket
```

Comment:
shouldnt you skip this check if there is a binary override url?

Reply:
i didn't wanna touch the url override functionality as that can be used in CI. i only added support for file override from the filesystem and that should still check if the binary exists because the kots version will be the sha of the binary file and should not re-upload if already exists